src/natcap/invest/sdr/sdr_core.pyx

```diff
@@ -1,5 +1,6 @@
 # cython: profile=False
-# cython: language_level=3
+# cython: language_level=2
```

Comment:
I'm curious why this would become 2? Is that indicating the Python language level of 2?

Reply:
Good question, I changed this while experimenting with different settings and meant to change it back. From the cython docs,
> language_level (2/3/3str) Globally set the Python language level to be used for module compilation. Default is compatibility with Python 3 in Cython 3.x and with Python 2 in Cython 0.x. To enable Python 3 source code semantics, set this to 3 (or 3str) at the start of a module or pass the “-3” or “–3str” command line options to the compiler. For Python 2 semantics, use 2 and “-2” accordingly. The 3str option enables Python 3 semantics but does not change the str type and unprefixed string literals to unicode when the compiled code runs in Python 2.x. Language level 2 ignores x: int type annotations due to the int/long ambiguity. Note that cimported files inherit this setting from the module being compiled, unless they explicitly set their own language level. Included source files always inherit this setting.

So I'll change it back to 3 for now, and when we update to cython 3, we might want to remove it entirely.